server/lostandfound/urls.py

```diff
@@ -19,4 +19,5 @@
 admin.site.site_header = "Lost and Found Administration"
 urlpatterns = [
     url(r'^admin', admin.site.urls),
+    url(r'^admin/', admin.site.urls),
```

Comment:
It seems it only works for the default route to the index page `localhost:8000`, because with the value of `APPEND_SLASH` set to `True`, `localhost:8000/admin/` will still return a `404 status code`